code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -19,6 +19,10 @@
 	prime()
 	..()
 
+/obj/item/weapon/grenade/acid_melt()
+	prime()

```

Comment:
hey woah! Is this new or replicating existing functionality?

Could be interesting